### ISSUE TITLE
Release v2.5.1

### ISSUE THREAD

**Issue by narqo** [bem/bem-components#1790]
After #1783 will be merge, if'd be nice to have a bugfix release of bem-components.

**Comment by narqo** [comment]
@tadatuta may I ask you to do this?

**Comment by tadatuta** [comment]
Released main version but not closing till I release `dist` to `bower` and CDN.

**Comment by narqo** [comment]
Still haven't?

**Comment by tadatuta** [comment]
https://yastatic.net/bem-components/2.5.1/desktop/bem-components.dev.js+bemhtml.js

https://github.com/bem/bem-components-dist/releases/tag/v2.5.1
